src/main/scala/swiss/dasch/api/ProjectsEndpointsHandler.scala

```diff
@@ -73,6 +76,24 @@ final case class ProjectsEndpointsHandler(
             ),
     )
 
+  private val deleteProjectsEraseEndpoint: ZServerEndpoint[Any, Any] = projectEndpoints.deleteProjectsErase
+    .serverLogic(userSession =>
+      shortcode =>
+        authorizationHandler.ensureAdminScope(userSession) *>
+          projectService.findProject(shortcode).some.mapError(projectNotFoundOrServerError(_, shortcode)) *> {
+            if (features.allowEraseProject) {
+              projectService
+                .deleteProject(shortcode)
+                .mapBoth(
+                  InternalServerError(_),
+                  _ => ProjectResponse.from(shortcode),
+                )
+            } else {
+              ZIO.fail(Unauthorized("Project erasure is not enabled."))
```

Comment:
Question: According to the error message, should this be `Unauthorized` error type or rather something different? If you decide to apply above method name change, consider also the adjustment of the message to something like `Permanent deletion of projects is not enabled.` This should be rather plural, because we allow deletion of all projects or not at all.

Reply:
Maybe `Forbidden` is more suitable; I am happy for any suggestion of a better http status code?

Reply:
How about: https://github.com/dasch-swiss/dsp-ingest/pull/218/commits/0bc2dcea7a7f78b3920c589b270dddb4600a2df4 ?